ci/terraform-taint/jobs/dev-taint.yml

```diff
@@ -0,0 +1,26 @@
+jobs:
+  - name: development-terraform-taint
+    max_in_flight: 1
+    plan:
+      - get: aws-ingestion
+        trigger: false
+      - get: aws-sdx
+        trigger: false
```

Comment:
aggregate gets

Reply:
Does `aggregate` support having only one step? 
This job is meant to be as simply adaptable to different scenarios as possible, in many of those it will only need one resource and one taint task. If adding `aggregate` means next time the whole `aggregate` step will have to be edited out I believe it would provide more scope for error.
Performance benefit of using aggregate is unimportant considering use cases for this job.

Reply:
Yeah, aggregate supports having a single step in it. It's not purely performance, if one item failed (e.g. for a state lock) you'd probably still want it to do the others

Reply:
Considering the word "probably" in your phrase above and the danger this pipeline poses should the approach be more cautious along the lines  of "if any `get` step has failed a user error is likely in the job, do not proceed"?

Reply:
if any `get` step fails then it can continue with the other `get`s, but it won't run the tasks. If any of the `task`s fail, it would be cleaner to continue